src/content/docs/logs/forward-logs/heroku-log-forwarding.mdx

```diff
@@ -54,47 +101,31 @@ To enable our log management capabilities, start by creating a Heroku Syslog dra
    }
    ```
 
-## Register a Heroku syslog drain [#register-syslog]
-
-Next, you'll need to register your newly created Heroku Syslog drain in New Relic:
-
-1. Log in to [New Relic's Logs UI](https://one.newrelic.com/launcher/logger.log-launcher) and click **Add more data sources**.
-
-   <img
-     title="Add more data sources button"
-     alt="Add more data sources button"
-     src={addData}
-   />
+### Register a Heroku syslog drain [#register-syslog]
 
-2. Click the **Heroku** tile under **Log ingestion**.
+Next, you will need to register your newly created Heroku Syslog drain in New Relic:
 
-   <img
-     title="Heroku tile"
-     alt="Heroku tile"
-     src={herokuTile}
-   />
-
-3. Choose the New Relic account to stream your Heroku application logs to, and click **Continue**.
-
-4. Paste your newly created Heroku drain token in the **Heroku drain token** field.
-
-5. Click **Add Heroku drain log** to complete registration.
+1. Make sure your New Relic user account has the [admin role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#roles) assigned to it.
+2. Go to the [New Relic Marketplace](https://one.newrelic.com/marketplace).
+3. Under **Logging**, click the `Heroku (syslog)` tile. You can also search for it using the search bar.
+4. If you have multiple accounts, select the account in which you want to register the drain token.
+5. Paste your newly created Heroku drain token in the **Heroku drain token** field.
+6. Click **Add Heroku drain log** to complete the registration.
 
 <Callout variant="important">
   Heroku currently doesn't support customizing the format of logs sent from Logplex. For more information, see [Heroku's log format documentation](https://devcenter.heroku.com/articles/logging#log-format).
 </Callout>
 
-## Delete a Heroku syslog drain token mapping [#delete-syslog-drain]
+### Delete a Heroku syslog drain token mapping [#delete-syslog-drain]
 
 Complete these steps to delete a Heroku syslog drain token mapping via the New Relic UI.
```

Comment:
@danybmx Could you add an introductory sentence here telling users what to expect in this section? How about something like this: "You can remove drains using the New Relic UI or with the REST API."